controllers/database_reconciler_helpers.go

```diff
@@ -70,15 +70,19 @@ func (r *DatabaseReconciler) addFinalizer(ctx context.Context, req ctrl.Request,
 func (r *DatabaseReconciler) handleDelete(ctx context.Context, database *ndbv1alpha1.Database, ndbClient *ndb_client.NDBClient) (ctrl.Result, error) {
 	log := ctrllog.FromContext(ctx)
 	log.Info("Database CR is being deleted.")
+	r.recorder.Event(database, "Normal", "DatabaseCRDeletionStarted", "Database CR is being deleted.")
```

Comment:
Maybe you can put "Database CR is being deleted" in an errMessage statement like you did else where.

Reply:
Done